src/components/layout.css

```diff
@@ -0,0 +1,622 @@
+html {
```

Comment:
Where is this CSS file from? It has a lot of styles in it that we're not using (yet), is it from a framework? I'd prefer to use something well-established that we can theme...

Reply:
Yes it is from the `gatsby` (the default config)

I will clean it on the next PR

Reply:
No worries, I'm fine starting out with a totally default configuration - just wanted to know what we're getting here ;-)